docs/script/test

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+## This script is used for testing on Jenkins @ ci.deis.io
+
+set -e # halt script on error
+set -x
+
+build_dest=$1
+
+id
+ls -ld ${build_dest}
+
+cd /app
+
+echo "Building site..."
+/src/script/build ${build_dest}
+
+echo "Checking links..."
+bundle exec htmlproofer ${build_dest} --only-4xx --check-html --disable-external --file-ignore "/contact\/index\.html/" || exit 0
```

Comment:
currently won't run `htmlproofer` b/c we don't install it for docs (no Gemfile) -- can be follow-up TODO (might even be rad to bake the util into gutenberg for use here so we don't need to be Gemfile-ing)